api/v1/db/oracledb/player-dao.js

```diff
@@ -65,12 +66,57 @@ const getPlayerByGameIdAndPlayerId = async (id, pid) => {
     if (_.keys(groupedRawPlayers).length > 1) {
       throw new Error('Expect a single object but got multiple results.');
     } else {
-      const serializedPlayer = serializePlayer(rawPlayers, id);
+      const serializedPlayer = serializePlayer(rawPlayers, id, isPost);
       return serializedPlayer;
     }
   } finally {
     connection.close();
   }
 };
 
-module.exports = { getPlayersByGameId, getPlayerByGameIdAndPlayerId };
+const postPlayerByGameId = async (body, gameId) => {
+  const connection = await conn.getConnection();
+  try {
+    body = body.data.attributes;
+    body.outId = {
+      type: oracledb.NUMBER,
+      dir: oracledb.BIND_OUT,
+    };
+
+    const { playerCards } = body;
+    delete body.playerCards;
+    const postSqlQuery = `
+    INSERT INTO PLAYERS (MEMBER_ID, GAME_ID, PLAYER_BET, STATUS_ID)
+    VALUES (:memberId,
+           ${gameId},
+           :playerBet,
+           (SELECT STATUS_ID FROM STATUSES WHERE STATUSES.STATUS = :playerStatus)) RETURNING PLAYER_ID INTO :outId
```

Comment:
Could you move the `RETURNING ...` statement to a new line since this follows the `VALUES` section?